sentry-android-core/build.gradle.kts

```diff
@@ -64,6 +64,11 @@ android {
         // We run a full lint analysis as build part in CI, so skip vital checks for assemble tasks.
         isCheckReleaseBuilds = false
     }
+
+    // needed because of Kotlin 1.4.x
```

Comment:
What is this for? Besides I assume getting to compile

Reply:
we use `org.jetbrains:annotations:20.1.0` and they pinned `13.0.0` but the API has only addition to the lib, so it's not an issue to overwrite it.